pom.xml

```diff
@@ -51,7 +51,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.3.3.RELEASE</version>
+        <version>2.3.4.RELEASE</version>
```

Comment:
It looks like there are already dependabot pull requests for this updates.  If I was a purist, I would say these
dependency updates do not belong in this PR request, since they really are not related to the issues/change in
the Jira issue. On the other hand, it is a very small update and what really matters is that they get done. 
If they are kept here, the dependable pull requests should be closed. Dependabot will open another for the next update, such a spring 2.3.5  ... Either way is fine with me.